### PR TITLE
YLP-515 Update note (message) code for Blip v2

### PR DESCRIPTION
**Pull request by yvon-dblg**

### CODE REVIEW

**Review by franck-fourel — CHANGES_REQUESTED** [review]
The new note icon does not create a new note. Here is the message in the console
image

**Comment by franck-fourel** [comment]
I also got a Logout in the middle of a session
here is a screenshot of what I got. I was still able to navigate but I was not able to get patient data
image

**Comment by yvon-dblg** [comment]
> 
> 
> The new note icon does not create a new note. Here is the message in the console

Fixed in https://github.com/mdblp/blip/pull/427

**Comment by yvon-dblg** [comment]
> 
> 
> I also got a Logout in the middle of a session
> here is a screenshot of what I got. I was still able to navigate but I was not able to get patient data

It was another problem: The server config was not right

But you have a point, we need to manage this situation
I will propose a PR for that later

**Review by franck-fourel — APPROVED** [review]
LGTM

PR https://github.com/mdblp/blip/pull/427 will fix the found issue.

File: packages/tideline/js/tidelinedata.js [hunk 1]
```diff
@@ -14,1010 +14,1090 @@
...
-    startTimer('adjustFillsForTwoWeekView');
-    var fillData = this.grouped.fill;
-    var endpoints = getTwoWeekFillEndpoints();
-    this.twoWeekData = this.grouped.smbg || [];
-    var twoWeekFills = fillDataFromInterval(new Date(endpoints[0]), new Date(endpoints[1]), false);
-    this.twoWeekData = _.sortBy(this.twoWeekData.concat(twoWeekFills), function(d) {
-      return d.normalTime;
+  if (!Array.isArray(this.grouped.pumpSettings)) {
+    return defaultPumpManufacturer;
+  }
```
Comment (franck-fourel) [inline packages/tideline/js/tidelinedata.js:651]: @yvon-dblg 
This has to be fixed in later version. 
If we have no settings (I know that should not happen) the default pump manufacturer is not added to the deviceEvents. At the end the reservoir  tooltip is broken.

**Pull request merged**